docs/mkdocs.yml

```diff
@@ -43,6 +43,7 @@ nav:
   - how-to/external-dns.md
   - how-to/metrics-sets.md
   - how-to/node-tuning.md
+  - how-to/hibernate-workers.md
```

Comment:
Let's instead a new section: "Automated Machine Management" and let's this be a subsection of it, e.g. "Scaling down data plane to Zero"